### PR TITLE
Update add-licenses-using-product-key.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as the options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9258

File: microsoft-365/commerce/licenses/add-licenses-using-product-key.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ Prepaid licenses are issued to you as a 25-character alphanumeric code. After yo
 ::: moniker-end
 
 2. On the **Products** tab, select the subscription to which you want to add licenses.
-3. On the subscription details page, in the **Licenses** section, select **Add more licenses**.
-4. In the **Add more licenses pane**, select **Use a new and unused product key**, then select **Next**.
-5. Enter the product key, then select **Next**.
+3. On the subscription details page, in the **Licenses** section, select **Add  licenses**.
+4. In the **Add licenses pane**, select **Redeem Product Key**
+5. Enter the product key, then select **Validate** and **Next**
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/licenses/add-licenses-using-product-key.md]:
```suggestion
3. On the subscription details page, in the **Licenses** section, select **Add licenses**.
4. In the **Add licenses pane**, select **Redeem Product Key**.
5. Enter the product key, then select **Validate** and **Next**.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by cmcatee-MSFT** [comment]
Article has been retired and the procedures moved to 2 separate articles. After I review the UI with the PM, I'll update the necessary files. In the meantime, closing this PR.

**Pull request closed without merging**